README.rst

```diff
@@ -0,0 +1,77 @@
+pyincore
+========
+
+**pyIncore-incubator** is a component of IN-CORE. It is a python package
+consisting of new analyses that build on pyIncore, but are not yet part of the
+core analyses.
```

Comment:
maybe add a sentence to explain its connection with pyincore?

Reply:
Something like this? If not, can you suggest what's missing and I can try to rework the text? Thanks!

**pyIncore-incubator** is a component of IN-CORE. It is a python package
consisting of new analyses that build on pyIncore, but are not yet part of the 
core analyses. Analyses extend the BaseAnalysis class from pyIncore and add new
functionality.